docs/guide/search/extending_search/custom_search.md

```diff
@@ -0,0 +1,79 @@
+# Implement custom search
+
+This example shows how to create a new service which searches for products that:
+
+- are discontinued
+- are visible
+
+It sets the sorting option to `relevance`, adds a Manufacturer facet,
+and boosts some of the fields (they will be more relevant than others).
+
+The built-in implementation of search interface has all methods required for searching using the Solr gateway.
+
+To get discontinued products the controller uses `SearchTermCondition`, which has the search phrase and the specific field.
+In this case, in the current data model, all discontinued fields have the word `DECLINE` in Solr field `ses_product_ses_discontinued_value_s`.
+A condition for visibility ensures that only visible products are returned.
+
+Boosting is set up so that SKU and name are more important than long description.
+
+The field boosting condition is implementation dependent. This means that the field name values which are passed to the constructor are different for eContent and content model search.
```

Comment:
Consider adding a hyphen between implementation anddependent.
https://courses.lumenlearning.com/suny-styleguide/chapter/hyphens-and-dashes/